crypto-ffi/src/client_id.rs

```diff
@@ -27,7 +27,7 @@ impl ClientId {
         self.0.clone()
     }
 
-    pub(crate) fn from_cc(id: core_crypto::ClientId) -> ClientIdMaybeArc {
+    pub(crate) fn from_cc(id: core_crypto::ClientId) -> Arc<ClientId> {
         std::sync::Arc::new(ClientId(id))
```

Comment:
Could adjust this, now that you're importing `Arc`; could be just `Arc::new` without the full path.

Reply:
yes, this got removed in a later commit